js/skin.js

```diff
@@ -212,6 +212,7 @@ var Skin = React.createClass({
               responsiveView={this.state.responsiveId}
               componentWidth={this.state.componentWidth}
               videoQualityOptions={this.state.videoQualityOptions}
```

Comment:
same thing here.  Sorry for making you do this but we have enough problems with null checks. May as well fix ones we see.